nodes/src/nodes/llm_vision_gemini/IInstance.py

```
@@ -0,0 +1,115 @@
+
+    # Cached answer text from writeDocuments so writeImage can reuse it without
+    # a second Gemini call for the same frame.
+    _cached_answer: str = None
+
+    def writeImage(self, action: int, mimeType: str, buffer: bytes):
+        """Handle AVI image protocol for streaming image frames."""
+        if action == AVI_ACTION.BEGIN:
+            self.image_data = bytearray()
+            self._cached_answer = None
+            return self.preventDefault()
+        elif action == AVI_ACTION.WRITE:
```

Ruff (0.15.10)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)